examples/demo-subscription-with-logger.php

```diff
@@ -17,6 +17,7 @@
 use Amp\Success;
 use Prooph\EventStoreClient\Internal\EventStoreCatchUpSubscription;
 use Prooph\EventStoreClient\Internal\StopWatch;
+use Throwable;
```

Comment:
Perhaps extract and merge the convention changes into their own PR. Given the size of this PR it would make review easier.